eng/devices/android.cake

```diff
@@ -157,7 +158,9 @@ Task("Build")
 
 	if (USE_DOTNET)
 	{
-		SetDotNetEnvironmentVariables(DOTNET_PATH);
```

Comment:
I think this was just wrong since was setting the Dotnet path to the bin/dotnet/dotnet